src/CompDevSDKMain.c

```diff
@@ -0,0 +1,19 @@
+#include "CompDevSDK.h"
+
+#include "ports.h"
+
+TaskHandle_t BlinkTaskHandle;
+void BlinkTask(void *param);
```

Comment:
Zamiast forward declaration, mozesz od razu wstawic pelna deklaracje funkcji. Nie zagrozi to czytelnosci ani strukturze pliku.

Reply:
Ok.